packages/stack-shared/src/interface/crud/projects.ts

```diff
@@ -57,7 +57,7 @@ export const emailConfigSchema = yupObject({
 });
 
 const domainSchema = yupObject({
-  domain: schemaFields.projectTrustedDomainSchema.defined(),
+  domain: schemaFields.urlSchema.defined(),
```

Comment:
you are missing the `.meta` part from the projectTrustedDomainSchema now (it's required for the docs)